site/docs/configuration/expected-outputs/deterministic.md

```diff
@@ -261,6 +262,72 @@ assert:
     value: file://./path/to/schema.json
 ```
 
+### Is-SQL
+
+The `is-sql` assertion checks if the LLM output is a valid SQL statement.
```

Comment:
we should mention that the user needs to `npm install node-sql-parser`

Reply:
got it